Translators/Signs/vsl_tim_generator.py

```diff
@@ -0,0 +1,17 @@
+from itis_codes import ItisCodes
+
+def get_itis_codes():
+    return [ItisCodes.SPEED_LIMIT.value]
+
+def get_geometry(geometry):
+    if len(geometry) == 0:
+        return []
+    annotated_geometry = []
+    # Check if the geometry is a single point
+    if type(geometry[0]) == float:

```

Comment:
suggestion(non-blocking): if we pulled the "geometry" object instead of the "coordinates" here we could look to the type of "Point" instead. This might make this a bit more clear

Reply:
I've updated this to pull the 'geometry' object instead of coordinates